tasks.star

```diff
@@ -328,15 +211,6 @@ def configure():
         cmds = ["tool check-deps"],
     )
 
-    # This is necessary because VSCode's clangd extension only supports a single compile_commands.json at the root
-    # of the project.
-    merge_compile_commands = task(
-        "merge-compile-commands",
-        desc = "Merges all compile_commands.json files into one",
-        deps = ["build-tool"],
-        cmds = ["tool merge-compile-commands compile_commands.json build/*/compile_commands.json"],
-    )
-
     if OS == "windows":
         task(
             "bootstrap-mingw64",
```

Comment:
I think there might be a better home for this. Maybe inside `helpers.star` as this is entirely related to the cmake wrapper and the `bootstrap-mingw64` task name should be contained in one file, I think.